src/utils/link-object-extractor.ts

```diff
@@ -0,0 +1,230 @@
+import { Variation } from '../model';
+import { LinkBlock } from '../model/link-block';
+import { DocumentBodyBlock } from 'knowledge-html-converter/dist/models/blocks/document-body-block';
+import {
+  DocumentBodyParagraph,
+  DocumentContentBlock,
+} from 'knowledge-html-converter/dist/models/blocks/document-body-paragraph';
+import {
+  DocumentBodyImage,
+  DocumentBodyList,
+  DocumentBodyTable,
+  DocumentBodyTableCellBlock,
+  DocumentBodyTableRowBlock,
+  DocumentListContentBlock,
+  DocumentTableContentBlock,
+  DocumentText,
+} from 'knowledge-html-converter';
+import { DocumentBodyListBlock } from 'knowledge-html-converter/dist/models/blocks/document-body-list';
```

Comment:
Expose this class in html converter, and do not import anything from dep's dist folder.